composer.json

```diff
@@ -19,6 +19,9 @@
     "autoload": {
         "psr-4": {
             "PayU\\Alu\\": "src/PayU/Alu/",
+            "PayU\\Payments\\": "src/PayU/Payments/",
+            "AluV3\\": "src/PayU/Payments/Gateways/AluV3",
```

Comment:
Here we should have only one autoloader added: `"PayU\\Payments\\": "src/PayU/Payments/"`

Reply:
done